### ISSUE TITLE
10Gb ram allocated and crash at start 

### ISSUE THREAD

**Issue by HaXx4Real** [FTBTeam/FTB-Modpack-Issues#2820]
### Modpack

FTB Presents Direwolf20 1.19

### Modpack version

1.6.1

### Log Files

[hs_err_pid7208.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/10936523/hs_err_pid7208.log)

### Describe the bug

I am unable to play the modpack because it keeps crashing due to insufficient memory, I´ve tried on other modpacks and it does the same. Sometimes it opens and crashes while i try to join world . I allocated 4gb , 6gb , 8,9,10 and it does not seem to help the issue , i tried some jvm commands but didn´t work. Tried to uninstall java and install again and nothing, tried multimc too but no help either.  I got to play a little without crashing the first time i opened despite some low fps ( ~25 ). I have no clue why it keeps crashing on start and even with other older minecraft versions modpacks like ultimate reloaded. This is my system specs :
-windows 11 pro
-AMD Ryzen 7 5700U with Radeon Graphics      1.80 GHz 
-16gb ram 

Any help is appreciated !

### Steps to reproduce

1-Open modpack
2-Watch it crash due to insufficient memory 

### Expected behaviour

Expect to find an answer and get to play ftb 

### Screenshots

![Captura de ecrã 2023-03-09 220516](https://user-images.githubusercontent.com/127449016/224174362-49f89423-7dfd-41d8-9f66-ec7fc5a0042e.png)

**Comment by HaXx4Real** [comment]
Apparently somehow i set no paging file probably months ago, set it to manage automatically and it doesnt crash anymore.

**Comment by CrazyPoopy** [comment]
Hi, I'm having the same exact problem you were having, but I don't understand what you meant by setting no paging file. Please help me fix this, my discord is ImPoopy. Thanks

**Comment by HaXx4Real** [comment]
Hi, i just saw your email and I dont remember the github account soo im
answering here, you can look up on google how to set up manage paging file
automatically, i found this article soo you can know :
https://www.tomshardware.com/news/how-to-manage-virtual-memory-pagefile-windows-10,36929.html

[image: imagem.png]
Just select first box ( automatically manage paging file size for all
drivers ) if you have this case. It was my problem and it fixed it, hope it
helps


CrazyPoopy ***@***.***> escreveu no dia sexta, 29/12/2023
à(s) 02:09:

**Comment by CrazyPoopy** [comment]
Hey, thank you so much for getting back to me, but my cousin and I found
out that the gravestones were causing our ram to go up until we crashed,
even if we left the server our ram wouldn’t change. So I went in creative
and broke them (there were like three) and that fixed it completely. 3-5
gigs of ram is whats pretty average for me now. Anyways, thanks for
everything
Reply to this email directly, view it on GitHub
<https://github.com/FTBTeam/FTB-Modpack-Issues/issues/2820#issuecomment-1875562700>,
or unsubscribe
